chat/src/Components/ChatMessageForm.svelte

```diff
@@ -321,6 +529,57 @@
         </div>
     {/if}
 
+    {#each [...$applicationsSelected] as app}
+        <div
+            class="tw-mx-2 tw-mb-2 tw-px-6 tw-py-3 tw-flex tw-flex-wrap tw-bg-dark-blue/95 tw-rounded-xl tw-text-xxs tw-justify-between tw-items-center tw-bottom-12"
+        >
+            <div class="tw-flex tw-flex-row tw-justify-between tw-items-center tw-m-1 tw-w-full">
+                <label for="app" class="tw-m-0">
+                    <img src={app.icon} alt={app.name} width="20px" />
+                    {#if app.name === "Klaxoon"}
+                        {$LL.form.application.klaxoon.description()}
+                    {/if}
+                    {#if app.name === "Youtube"}
+                        {$LL.form.application.youtube.description()}
+                    {/if}
+                    {#if app.name === "Google Docs"}
+                        {$LL.form.application.googleDocs.description()}
+                    {/if}
+                    {#if app.name === "Google Sheets"}
+                        {$LL.form.application.googleSheets.description()}
+                    {/if}
+                    {#if app.name === "Google Slides"}
+                        {$LL.form.application.googleSlides.description()}
+                    {/if}
```

Comment:
This looks suspiciously hard to maintain. Maybe the "app" object could contain a pointer to the `$LL.form.application[app_key]` translation?